Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM nvidia/cuda
+
+# Install miniconda
+RUN apt-get update && apt-get install -y wget
+RUN MINICONDA="Miniconda2-latest-Linux-x86_64.sh" && \
+    wget --quiet https://repo.continuum.io/miniconda/$MINICONDA && \
+    bash $MINICONDA -b -p /miniconda && \
+    rm -f $MINICONDA
+ENV PATH /miniconda/bin:$PATH
+
+# Add omnia
+RUN conda config --add channels omnia
+
+# Install deepchem
+RUN conda install --yes deepchem
```

Comment:
@jchodera there is want for a stable alpha release https://github.com/deepchem/deepchem/issues/376

Currently users must install the code from source following the instructions in README.md (although the easy install script makes all the conda dependencies much easier /scripts/install_deepchem_conda.sh)

Another option would be to pin the git checkout to head today (415aebadff54175b7ba108964723c8f69438af94), install that in the docker image and then update when alpha is released.